sdk/src/test/java/com/mapbox/maps/ResourceOptionsManagerTest.kt

```diff
@@ -60,5 +105,15 @@ class ResourceOptionsManagerTest {
       ResourceOptionsManager.getDefault(context).resourceOptions.tileStoreUsageMode
     )
     assertEquals("baseUrl", ResourceOptionsManager.getDefault(context).resourceOptions.baseURL)
+
+    ResourceOptionsManager.destroyDefault()
+    defaultResourceOptionsManager = ResourceOptionsManager.getDefault(context)
+    assertEquals("token", defaultResourceOptionsManager.resourceOptions.accessToken)
+    Assert.assertTrue(defaultResourceOptionsManager.resourceOptions.cachePath!!.endsWith("/.mapbox/maps/ambient_cache.db"))
+    assertEquals(DEFAULT_CACHE_SIZE, defaultResourceOptionsManager.resourceOptions.cacheSize)
+
+    ResourceOptionsManager.destroyDefault()
+    defaultResourceOptionsManager = ResourceOptionsManager.getDefault(context)
+    assertEquals("token", defaultResourceOptionsManager.resourceOptions.accessToken)
```

Comment:
Seems redundant?

Reply:
It's to verify the regenerated ResourceOptionsManager

Reply:
Yeah, this part could be deleted.